test/bases/renku_data_services/data_api/test_sessions.py

```diff
@@ -976,40 +1043,6 @@ async def test_starting_session_anonymous(
     assert session_res.json["name"] in [i["name"] for i in res.json]
 
 
-@pytest.mark.asyncio
-async def test_post_build(sanic_client: SanicASGITestClient, user_headers, create_project) -> None:
```

Comment:
Why is this test removed? We still need to check that triggering a build is working as expected.

Reply:
We start the build when the session launcher is posted and starting the build again will fail. We can stop this build and then rebuild it again but that's what we exactly do in `test_get_build`. I'll re-introduce this test with some modification to at least keep the action name in the test.